server/pbench/bin/pbench-sync-satellite

```diff
@@ -78,17 +78,36 @@ else
     dryrun=
 fi
 
+# check whether any previous ssh failure; if any try again and if it fails again then exit without going further.
+if [ -s $LOGSDIR/change_state.$prefix.log ] ;then
+    ssh $remotehost "cd $remotearchive; /opt/pbench-server/bin/pbench-satellite-state-change" < $LOGSDIR/change_state.$prefix.log
+    status=$?
+    if [[ $status != 0 ]] ;then
+        echo "Unable to change the state in $remotehost" | mailx -s \
+        "FAILED: $PROG: $prefix: ssh to $remotehost failed twice"\
+        $mail_recipients
+        exit 1
+    else
+        rm $LOGSDIR/change_state.$prefix.log
+        status=$?
+        if [[ $status != 0 ]]; then
+            echo "Failed: rm $LOGSDIR/change_state.$prefix.log twice"
```

Comment:
"Twice" is  right for the ssh error message, but it is wrong here: the ssh failed before (hence the file is still around) and now it passed (hence we try to remove the file for the *first* time).